.github/runTests.sh

```diff
@@ -0,0 +1,17 @@
+echo $PWD
+for file in $1
+do
+        echo $file
+        filename="$(basename -- $file)"
+        extension="${filename##*.}"
+        parentdir="$(basename "$(dirname "$file")")"
+        echo parent_dir:${parentdir}
+        if [[ "$parentdir" =~ ^(dblp|spotify|twitter|yelp)$ ]]; then
```

Comment:
Could it be more general? Currently, this requires us to change the script every time when there's a new config coming in.

Reply:
changed to search subdirectories under root of repo